torchtune/models/llama4/_component_builders.py

```diff
@@ -180,6 +181,7 @@ def llama4_decoder(
     num_experts: int = 16,
     experts_per_token: int = 1,
     use_shared_expert: bool = True,
+    use_grouped_mm: bool = True,
```

Comment:
Do we actually need to expose this as an option? Are there cases where it's supported and not the fastest solution?